components/automate-chef-io/data/docs/api-static/02-meta-description.yaml

```diff
@@ -1,14 +1,41 @@
 info:
   description: |
     # Authentication
-    The standard way of authenticating for the Automate API is to generate an
-    API token and pass that with your API requests with the `api-token` header.
-    An example request using curl looks like this:
+    The Chef Automate API typically uses am API token and passed in the header of  your API request.
+
+    To create admin token can and set it as an environment variable use the following command:
+
+    ```
+    export TOKEN=`chef-automate admin-token`
+    ```
+
+    An admin token has unlimited access the entire Chef Automate API.
+
+    Pass the token as part of the API call. For example:
 
     ```
     curl -s -H "api-token: $TOKEN" https://automate.example.com/api/v0/auth/policies -v
     ```
 
-    For more information about authentication, refer to the
-    [API Tokens](https://automate.chef.io/docs/api-tokens/)
-    section of the Automate documentation.
+    To create api tokens with limited permissions, use your admin token to create a standard token. You can then write a policy that exactly defines the API access for the standard token.
+
+    In this example, the `curl` command creates the new token and the `export` command saves as an environment variable with the name "TOKEN2".
```

Comment:
I'm not convinced we should keep this, because we don't explain how to set permissions for TOKEN2. Still, it might be helpful & we won't know unless we publish it.

Reply:
i think its okay to keep for now to at least communicate that you can define different permissions for tokens.. this is a lot easier in iam v2 as well. so we can revisit after the force upgrade